src/FormattingSettingsComponents.ts

```diff
@@ -397,7 +402,10 @@ export class ItemFlagsSelection extends SimpleSlice<string> {
     }
 }
 
-export class AutoFlagsSelection extends SimpleSlice<powerbi.EnumMemberValue> {
+/**
+ * Multiple flags selection component with enumeration values defined in capabilities.json using bitwise values.
```

Comment:
not a bitwise

Reply:
Added clarifiaction to JSDoc comment. API still retuns a single number representing a combination of all received flags.